src/NServiceBus.ServicePlatform.Connector/NServiceBus.ServicePlatform.Connector.csproj

```diff
@@ -16,12 +16,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="[7.0.1,8)" />
-    <PackageReference Include="NServiceBus.Heartbeat" Version="3.0.1" />
-    <PackageReference Include="NServiceBus.CustomChecks" Version="3.0.1" />
-    <PackageReference Include="NServiceBus.SagaAudit" Version="3.0.1" />
-    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="3.0.6" />
-    <PackageReference Include="NServiceBus.Metrics" Version="3.0.0" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.1901, 9.0.0)" />
+    <PackageReference Include="NServiceBus.Heartbeat" Version="[4.0.0-alpha.253, 5.0.0)" />
+    <PackageReference Include="NServiceBus.CustomChecks" Version="[4.0.0-alpha.253, 5.0.0)" />
+    <PackageReference Include="NServiceBus.SagaAudit" Version="[4.0.0-alpha.251, 5.0.0)" />
+    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="[4.0.0-alpha.164, 5.0.0)" />
+    <PackageReference Include="NServiceBus.Metrics" Version="[4.0.0-alpha.153, 5.0.0)" />
```

Comment:
We had all of these deliberately set to specific versions and not ranges. Otherwise customers taking dependencies on this package have to update those specifically (or they get the minimum version specified). We deemed that there wasn't a lot of risk in this approach because these downstream plugins don't update very frequently.

Reply:
I forgot about the min version thingy. I have set this to specific versions and removed the ranges.